extensions/timers/02_performance_util.js

```diff
@@ -0,0 +1,127 @@
+// Copyright 2018-2021 the Deno authors. All rights reserved. MIT license.
```

Comment:
This code is all duplicated with `webidl` extension / `Deno.core.serialize` / `Deno.core.deserialize`. The whole file could be removed with minor changes to performance.js.